File: 242-valid-anagram/242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] cnt = new int[26];
        if(s.length() != t.length()) return false;
        for(int i = 0; i < s.length(); i++) {
            cnt[s.charAt(i) - 'a'] += 1;
            cnt[t.charAt(i) - 'a'] -= 1;
        }
        for(int val : cnt){
            if(val != 0) return false;
        }
        return true;
    }
}

File: 242-valid-anagram/242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        cnt = [0]*26
        if len(s) != len(t):
            return False
        for i in range(len(s)):
            cnt[ord(s[i]) - ord('a')] += 1
            cnt[ord(t[i]) -ord('a')] -= 1
        for num in cnt:
            if num:
                return False
        return True